tests/test_batch_gap_fit.py

```diff
@@ -0,0 +1,27 @@
+import os, shutil, sys
+
+import numpy as np
+import pytest
+
+
+def test_batch_gap_fit():
```

Comment:
Should this be protected by tests for `gap_fit` exitsting, like in `tests/test_gap_fitting_simple.py`